src/rt-app_args.c

```diff
@@ -23,36 +23,79 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #include <stdio.h>
 #include <string.h>
 #include <sys/stat.h>
+#include <getopt.h>
 
 #include "rt-app_parse_config.h"
 
+char help_usage[] = \
+"Usage: rt-app <taskset.json>\n"
```

Comment:
What about reporting the version also here, before this short usage, to generate an output similar to this:

```bash
$> ./src/rt-app
rt-app v1.0-29-gd04f322 (2017-12-08 10:37:49 GMT)

Usage: rt-app <taskset.json>

Try 'rt-app --help' for more information.

$>
```

version should probably be added at L87

Reply:
I decided not to add it because many popular tools provide the version only when asked.

Reply:
I agree with balsini, that we don't want to pollute rt-app output unnecessarily with version each time it's executed